api/Document.json

```diff
@@ -1730,7 +1730,7 @@
                 "notes": "For backwards compatibility, the <code>options</code> argument can be an object or a string with the custom element tag name, although the string version is deprecated."
               },
               "chrome_android": {
-                "version_added": false,
+                "version_added": true,
```

Comment:
This shipped in 56: https://storage.googleapis.com/chromium-find-releases-static/fc1.html#fc17d505758c284f99624ac69b6e2fb72ea7b623

Also, `chrome`, `chrome_android`, and `webview_android` should be identical for this.

Reply:
Thank you!  I’ve added “56” as the version number for all Chromium-based browsers.